con-base/src/test/java/ba/codecentric/base/validation/FacilityTest.java

```diff
@@ -19,7 +18,6 @@
 
 @RunWith(SpringRunner.class)
 @SpringBootTest
-@SpringBootConfiguration
 public class FacilityTest {
```

Comment:
Sorry, it is not failing. Problem was on my side. But still we don't need those annotations.